jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/PostComment.java

```diff
@@ -152,4 +153,22 @@ public void putAttribute(String attributeName, String attributeValue) {
     public void putAllAttributes(Map<String, String> attributes) {
         this.attributes.putAll(attributes);
     }
+
+    /**
+     * Get a sign of delete
+     *
+     * @return true if the comment has been deleted
+     */
+    public Boolean getDeleted() {
```

Comment:
For auditing purposes.. We'll know when it happened. In the future we might add the author.. 

Or we could try using Envers right now: http://docs.jboss.org/envers/docs/ This helper-framework for Hibernate will manage deleting/editing logic on its own. But I have no experience with it, so can't say whether it has hidden issues.